data/css/modules/contentGroup/_contentGroup.scss

```diff
@@ -1,68 +1,54 @@
-.search-page-suggestions {
-    margin: 50px 0;
-    padding-bottom: 20px;
-
-    .ContentGroupContentGroup__title {
+.ContentGroupContentGroup {
+    &__title,
+    &__title--clickable {
         font-family: $display-font;
         font-weight: bold;
-        font-size: 40px;
-        color: $black-60-color;
-        padding-bottom: 30px;
+        font-size: 50px;
+        margin-right: 10px;
     }
-}
 
-.highlights, .hierarchical-set {
-    margin-top: 50px;
-    .ContentGroupContentGroup__trigger {
-        margin-bottom: 15px;
+    &__title {
+        color: transparentize(black, 1 - 0.60);
+    }
+
+    &__trigger {
         font-family: $support-font;
-        font-size: 20px;
-        color: $accent-light-color;
-        /* FIXME: We want a 'text-transform: lowercase;' here once GTK
-         * CSS supports such a property, or once we figure out Dynamic
-         * Titles in T11907
-         */
+        font-size: 25px;
+        margin-left: 10px;
 
         .ThemeableImage {
             -gtk-icon-source: $arrow-recolored-icon;
             min-height: 14px;
             min-width: 14px;
-            margin-left: 5px;
-            margin-bottom: 5px;
+            margin-left: 10px;
+            margin-bottom: 8px;
         }
+    }
 
-        &:hover {
-            color: $accent-dark-color;
-
-            .ThemeableImage {
-                -gtk-icon-source: $arrow-recolored-icon;
-            }
-        }
+    &__title,
+    &__title--clickable,
```

Comment:
If the BEM naming is correct, this isn't necessary; everything that has `.ContentGroupContentGroup__title--clickable` will also already have `.ContentGroupContentGroup__title`.